tests/testthat/test-renderer1-text.R

```diff
@@ -1,7 +1,16 @@
 acontext("Text")
 
 data(WorldBank, package = "animint")
-wb  <- WorldBank[WorldBank$year == 2010,]
+wb2010 <- subset(WorldBank, year==2010)
+subset(wb2010, population==min(population))
+### This test does not pass if we use the wb2010 data set above rather
+### than the wb data set below. The problem is that the scales are
+### trained before NAs are removed from the data. There are NA values
+### in life.expectancy/fertility.rate for Tuvalu, but not in
+### population. So Tuvalu will not be rendered on the plot, and in
+### fact there will be no text element with fontsize=10!
```

Comment:
did you know that scales are trained before removing NA? I did not! It results in some weird scales sometimes https://github.com/hadley/ggplot2/issues/1711